patches/components-permissions-android-BUILD.gn.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/components/permissions/android/BUILD.gn b/components/permissions/android/BUILD.gn
+index 3cfa933430a16ea55bfae73aa02e195a0aedd5a9..29ef945bf217fec6cf2af52a2f4bde7de00dfdda 100644
+--- a/components/permissions/android/BUILD.gn
++++ b/components/permissions/android/BUILD.gn
+@@ -14,6 +14,7 @@ generate_jni("jni_headers") {
+     "java/src/org/chromium/components/permissions/PermissionUtil.java",
+     "java/src/org/chromium/components/permissions/nfc/NfcSystemLevelSetting.java",
+   ]
++  import("//brave/components/permissions/android/java_sources.gni") sources += brave_components_permissions_java_jni_sources
```

Comment:
nit: we should probably move these imports to `brave/android/brave_java_sources.gni`, code will look cleaner

Reply:
tried that, but it doesn't seem right. I can't access our class from a chromium one directly, maybe the package name change and accessing it via full name will do the trick, but I don't think it will be cleaner.

Reply:
I meant just move this part `import("//brave/components/permissions/android/java_sources.gni")`

Reply:
got it. moved import to `brave/android/brave_java_sources.gni`